gen_binary.py

```diff
@@ -0,0 +1,66 @@
+from misc.viz_utils import visualize_instances_dict
+import numpy as np
+import json
+from skimage import io
+import cv2
+import zarr
+import sys, getopt, os
+import slideio
+
+'''
+Converts all json files in a folder to binary zarr.
+The corresponding svs files also need to be present to obtain height and width.
+All 3 files will have the same name.
+Sample call: python gen_binary.py -i /mnt/ibm_lg/snigdha/svs -o /mnt/ibm_lg/snigdha/svs
+'''
+def main(argv):
+	path = ''
+	output_path = ''
+	try:
+		opts, args = getopt.getopt(argv,"i:o:h:w:",["in_path=","out_path="])
+	except getopt.GetoptError:
+		print('gen_binary.py -i <in_path> -o <output_path>')
+		sys.exit(2)
+
+	for opt, arg in opts:
+		if opt in ("-i", "--in_path"):
+			path = arg
+		elif opt in ("-o", "--out_path"):
+			output_path = arg
+
+	for file in os.listdir(path):
+		# processing each json file
+		if file.endswith(".json"):
+			inputfile = os.path.join(path,file)
+			# no need to run the process if the binary zarr already exists
```

Comment:
what if the zarr created was incomplete?

Reply:
I don't think I have a way of checking that. A user inputted option to run the code again if zarr exists maybe?

Reply:
--rerun_zarr_creation perhaps